admin/handlers.php

```diff
@@ -188,8 +193,11 @@ function gigpress_error_checking($context) {
 				$errors['artist_name'] = __("You must enter an artist name.", "gigpress");
 			if($_POST['show_venue_id'] == 'new' && empty($_POST['venue_name']))
 				$errors['venue_name'] = __("You must enter a venue name.", "gigpress");
-			if($_POST['show_venue_id'] == 'new' && empty($_POST['venue_city']))
-				$errors['venue_city'] = __("You must enter a city.", "gigpress");
+			if ( true === $venue_city_required ) {
+				if($_POST['show_venue_id'] == 'new' && empty($_POST['venue_city'])) {
+					$errors['venue_city'] = __("You must enter a city.", "gigpress");
```

Comment:
```suggestion
					$errors['venue_city'] = __( 'You must enter a city.', 'gigpress' );
```

Reply:
Ideally, we should have the proper spacing and use the single quotes on these cases (if possible) to follow the WP coding standards: https://codex.wordpress.org/I18n_for_WordPress_Developers

Reply:
Again, fair point.
Was sticking to the original code standards, but you are probably right in whatever we touch maybe we should update.